### PR TITLE
correct verb

### PR DESCRIPTION
**Pull request by amyh**
Thank you so much for contributing code to Lando! If this is your **first time** contributing to Lando we recommend you check out

* [Contributing to Lando](https://docs.devwithlando.io/contrib/contributing.html)
* [Lando Developer Guide](https://docs.devwithlando.io/dev/started.html)

We will get to your PR as soon as we can but in the meantime you might want to check to make sure you have done the following. **The more boxes you can check the easier it will be for us to merge in your changes with confidence**

- [ X] My code includes the latest code from the `master` branch
- [ ] My code includes an update to the [CHANGELOG](https://github.com/lando/lando/tree/master/help)
- [ ] My code includes a [functional test](https://docs.devwithlando.io/dev/testing.html#functional-tests) if applicable
- [ ] My code includes a [unit test](https://docs.devwithlando.io/dev/testing.html#unit-tests) if applicable
- [ ] My code includes documentation updates if applicable.
- [ ] My code passes relevant CI status checks
- [ ] I have pinged [a project committer](https://docs.devwithlando.io/contrib/contributing.html#committers) when I think my code is ready for prime time.

### CODE REVIEW

File: plugins/lando-core/tasks/destroy.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = lando => {
       const app = lando.getApp(options._app.root);
       // Destroy the app
       if (app) {
-        console.log((chalk.green(`Preparing to resign ${app.name} to the dustbin of history...`)));
+        console.log((chalk.green(`Preparing to consign ${app.name} to the dustbin of history...`)));
```
Comment (uberhacker) [inline plugins/lando-core/tasks/destroy.js]: Actually, I think both verbs are fine.
```
resign: accept that something undesirable cannot be avoided.
consign: assign; commit decisively or permanently.
```

Comment (amyh) [reply plugins/lando-core/tasks/destroy.js]: No, that definition of resign only exists in the passive voice: for example, "I can't stand my cousin, but I _am resigned to_ her visit."
Consign implies a permanent and final disposition (source: https://www.merriam-webster.com/dictionary/consign, definition 2), which is what's being done with lando destroy. "Consign to the dustbin of history" also alludes to a specific quote of Leon Trotsky.

Comment (uberhacker) [reply plugins/lando-core/tasks/destroy.js]: But that is not what Trotsky quoted.  Here is what he said:
```
You are pitiful isolated individuals; you are bankrupts; your role is played out. Go where you belong from now on - into the dustbin of history!
```
What you argue is exactly why I hate the English language so much. :)

Comment (amyh) [reply plugins/lando-core/tasks/destroy.js]: If you'd quoted Trotsky in the Russian he actually spoke, I'd be more certain of your translation. Here's WIRED making use of the "consign" version, as one example: https://www.wired.com/2009/09/0904blackhole/.

And none of this contradicts my first two points, which are really the main ones. So why not quit your Googling and approve this? :)

Comment (uberhacker) [reply plugins/lando-core/tasks/destroy.js]: I'm not saying I disapprove.  I just think it's a petty difference not worth making a big fuss about. :)

Comment (pirog) [reply plugins/lando-core/tasks/destroy.js]: guys? WTF?

Comment (amyh) [reply plugins/lando-core/tasks/destroy.js]: The expression is "consign to the dustbin of history". If I get to the point where my app fails, I'm already stressed out, and seeing "resign" there just increses my stress. With thousands of users, I can't be the only one who feels this way. This change will make Lando look smarter.

**Comment by amyh** [comment]
I'm closing this pull request because I've submitted a new one based on the current state of the master branch. Thanks for your review.

**Pull request closed without merging**